components/style/core/base.less

```diff
@@ -61,6 +61,10 @@ a {
   cursor: pointer;
   transition: color .3s ease;
 
+  &:focus {
+    text-decoration: underline;
```

Comment:
Our designer don't wish underline on hover links. But it is focused state in this situation, I need to discuss with them later.